chain-impl-mockchain/src/transaction/witness.rs

```diff
@@ -78,7 +78,7 @@ impl WitnessAccountData {
         transaction_id: &TransactionSignDataHash,
         spending_counter: &account::SpendingCounter,
     ) -> Self {
-        let mut v = Vec::with_capacity(65);
+        let mut v = Vec::with_capacity(69);
```

Comment:
Similarly, I noticed we do `let mut v = Vec::with_capacity(65);` for `WitnessUtxoData` but this seems wrong since we're only writing 64 bytes (for some reason the tag byte isn't added)